crontabber/tests/test_crontabber.py

```diff
@@ -2149,6 +2150,170 @@ def test_job_list_separated_sloppily(self):
             ok_('Ran FooJob' in infos, infos)
             ok_('Ran BarJob' in infos, infos)
 
+    def test_run_two_jobs_simultaneously_first_time(self):
+        config_manager = self._setup_config_manager(
```

Comment:
For the record, that was not a valid idea. The problem was more profound than that.